response_operations_ui/templates/account/change-username.html

```diff
@@ -0,0 +1,93 @@
+{% set ns = namespace() %}
+
+{% set page_title = "Change username" %}
+{% block main %}
+{% include 'partials/flashed-messages.html' %}
+{% if errors %}
+  {% set errorTitle = 'There is 1 error on this page' %}
+    {% call
+            onsPanel({
+                "type": "error",
+                "classes": "u-mb-s",
+                "title":  errorTitle
+            })
+      %}
+<p xmlns="http://www.w3.org/1999/html">These <strong>must be corrected</strong> to continue.</p>
+            {% set errorData = [] %}
+            {% for error in errors %}
+            {% set error_text = error %}
+                {% do errorData.append(
+                    {
+                        "text": 'Problem with the ' ~ error_text.replace('_', ' '),
+                        "index": true,
+                        "url": "#" ~ error ~ '_error',
+                        "classes": "js-inpagelink"
+                    }
+                ) %}
+            {% endfor %}
+            {{
+                onsList({
+                    "classes": "list--bare",
+                    "itemsList": errorData
+                })
+            }}
+        {% endcall %}
+    {% endif %}
+<h1 class="u-fs-xl">Change username</h1>
+<p>This should match your windows login username</p>
+<form action="" method="post">
+        {{ form.csrf_token }}
+        {% if errors.username %}
+        {% set ns.errorUsernameText = NONE %}
+          {% for errorValue in errors.username %}
+            {% if ns.errorUsernameText == NONE %}
+              {% set ns.errorUsernameText = (errorValue) %}
+            {% else %}
+              {% set ns.errorUsernameText = ns.errorUsernameText~'</br>'~(errorValue) %}
+            {% endif %}
+          {% endfor %}
+            {% set errorUsername = { "text": ns.errorUsernameText,  "id": 'username_error' } %}
+          {% set username = form.username.data %}
+        {% endif %}
+            {{
+                onsInput({
+                    "id": "username",
+                    "name": "username",
+                    "type": "text",
+                    "label": {
+                        "text": 'Username'
+                    },
+                    "error": errorUsername,
+                    "value": username
+                })
+            }}
+
+    <p><br>You will need to sign in again after saving your changes.</p>
+
+
+    <div class="grid grid--flex grid--gutterless grid--vertical-center grid--no-wrap@s u-mt-m u-mb-m">
+        <div class="grid__col">
+            {{
+                onsButton({
+                "text": "Save",
+                "id": "btn-option-save",
+                "classes": 'btn-group__btn',
+                "submitType": "timer"
+                })
+            }}
+        </div>
+        <div class="grid__col u-ml-m btn-group__btn secondary">
+          <a href="{{ url_for('account_bp.get_my_account') }}" role="button" class="btn btn--link btn--secondary" id="btn-who-ons-option-cancel">
```

Comment:
that id for this link was obviously copied from somewhere else ;)  can you change it